### PR TITLE
docs: update guides

### LINKED ISSUE #1037: Rework guides

### PR DESCRIPTION
**Pull request by esoubiran-aneo**
fix #1037

Rename files in order to respect the filename conventions (https://aneoconsulting.github.io/armonik-docs-theme/guide/content-directory#filename), remove bold title becase a title is already important and move some content to others repo.

### CODE REVIEW

File: .docs/content/0.armonik/0.index.md [hunk 1]
```diff
@@ -1 +1,17 @@
...
+
+It provides a reference architecture that can be used to build and adapt a modern high throughput compute solution on-premise or using Cloud services, allowing users to submit high volumes of short and long-running tasks and scaling environments dynamically.
+
+::alert{type="info"}
+This project is an Open Source ([Apache 2.0 License](https://github.com/aneoconsulting/ArmoniK/blob/main/LICENSE)).
+::
+
+## When should I use ArmoniK
+
+ArmoniK should be used when the following criteria are meet:
```
Comment (lzianekhodja) [inline .docs/content/0.armonik/0.index.md:13]: ...following criteria are met:

File: .docs/content/2.guide/1.how-to/how-to-configure-seq.md [hunk 2]
```diff
@@ -1,13 +1,11 @@
-<!-- TODO: reorganize -->
-
-# Configure Seq
+# How to configure SEQ?
 
 If you want to access outside your kubernetes cluster, you have to configure seq to make it accessible from outside.
```
Comment (lzianekhodja) [inline .docs/content/2.guide/1.how-to/how-to-configure-seq.md:3]: Kubernetes cluster

Comment (lzianekhodja) [reply .docs/content/2.guide/1.how-to/how-to-configure-seq.md:3]: Seq

File: .docs/content/2.guide/1.how-to/how-to-configure-seq.md [hunk 3]
```diff
@@ -21,13 +19,13 @@ If you enabled the authentication of seq, the default credentials are the follow
...
+## Useful requests
 
 The application name for the control plane is `ArmoniK.Control`. (Application = 'ArmoniK.Control')
 The application name for the polling agent is `ArmoniK.Compute.PollingAgent`. (Application = 'ArmoniK.Compute.PollingAgent')
 The application name for the worker is `ArmoniK.Compute.Worker`. (Application = 'ArmoniK.Compute.Worker')
 
-# Troubleshoot
+## Troubleshoot
 
 If you use docker as the kubernetes backend, you should configure the logging driver of docker to `json-file`.
```
Comment (lzianekhodja) [inline .docs/content/2.guide/1.how-to/how-to-configure-seq.md:30]: Kubernetes backend

Comment (esoubiran-aneo) [inline .docs/content/2.guide/1.how-to/how-to-configure-seq.md:30]:
```suggestion
If you use docker as the Kubernetes backend, you should configure the logging driver of docker to `json-file`.
```

File: .docs/content/2.guide/1.how-to/how-to-configure-seq.md (hunk 2 above)
Comment (esoubiran-aneo) [inline .docs/content/2.guide/1.how-to/how-to-configure-seq.md:3]:
```suggestion
If you want to access outside your Kubernetes cluster, you have to configure Seq to make it accessible from outside.
```

File: .docs/content/0.armonik/0.index.md (hunk 1 above)
Comment (esoubiran-aneo) [inline .docs/content/0.armonik/0.index.md:13]:
```suggestion
ArmoniK should be used when the following criteria are met:
```

**Review by lzianekhodja — APPROVED** [review]

**Review by dbrasseur-aneo — APPROVED** [review]

**Review by aneojgurhem — APPROVED** [review]

**Pull request merged**